src/main/java/com/ghusse/ci/violations/bbcloud/lib/client/implementation/RestClient.java

```diff
@@ -32,39 +32,92 @@ public RestClient(NetHttpTransport transport, JsonHttpContentFactory contentFact
     this.requestFactory = this.transport.createRequestFactory();
   }
 
-  public InputStream get(String url) throws IOException {
-    HttpRequest request = this.requestFactory.buildGetRequest(new GenericUrl(url));
+  public InputStream get(String url) throws RestClientException {
+    HttpRequest request;
+    GenericUrl genericUrl = new GenericUrl(url);
+
+    try {
+      request = this.requestFactory.buildGetRequest(genericUrl);
+    } catch (IOException e) {
+      throw new RestClientException("Unable to create a delete request", "POST", genericUrl, e);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Unable to create a delete request" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)